cafebabel/templates/articles/detail.html

```diff
@@ -11,7 +11,17 @@
     {% endif %}
     <h1>{{ article.title }}</h1>
 
-    <p><time>Published on {{ article.creation_date.date() }}</time> — by {{ article.author }}<br>Reading time: {{ article.body|reading_time }} min</p>
+    <p>
+      <time>Published on {{ article.creation_date.date() }}</time>
```

Comment:
@cafebabel-webmaster FYI, the `time` tag should only enclose the… time ;)

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

I'm on it.